kubernetes/scripts/cluster-test.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Copyright 2021 CYBERCRYPT
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -euo pipefail
+
+export E2E_TEST_ADMIN_UAT=$(kubectl -n encryptonize exec -it deployment/encryptonize-deployment /encryption-service create-admin | grep "Access Token: " | awk '{print $(NF-1)}')
+make -C ../encryption-service credentials
```

Comment:
https://github.com/koalaman/shellcheck/wiki/SC2155:
```suggestion
E2E_TEST_ADMIN_UAT=$(kubectl -n encryptonize exec -it deployment/encryptonize-deployment /encryption-service 

export E2E_TEST_ADMIN_UAT
create-admin | grep "Access Token: " | awk '{print $(NF-1)}')
make -C ../encryption-service credentials
```